core/src/main/scala/org/apache/spark/status/api/v1/AllStagesResource.scala

```diff
@@ -69,7 +70,8 @@ private[v1] object AllStagesResource {
       }
 
     val taskData = if (includeDetails) {
-      Some(stageUiData.taskData.map { case (k, v) => k -> convertTaskData(v) } )
+      Some(stageUiData.taskData.map { case (k, v) =>
+        k -> convertTaskData(v, stageUiData.lastUpdateTime) } )
```

Comment:
Sorry I may stated clearly, it should be:

```
Some(stageUiData.taskData.map { case (k, v) =>
  k -> convertTaskData(v, stageUiData.lastUpdateTime) })
```
No whitespace after "}".

Reply:
Thanks for your patient.Actually i did not check carefully.